### ISSUE TITLE
Use mat or md prefix?

### ISSUE THREAD

**Issue by M4R1KU** [outcobra/outstanding-cobra#143]
Since the last update of @angular/material they are also using the "mat"-prefix` for everything as addition to the old "md"-prefix. They did this to make the new package more distinguishable to old angular material for AngularJS. Especially for applications that use both Angular 2 and AngularJS and @angular/material and the old angular material it is then much more easy to understand in which context they are (Angular 2 or AngularJS) and naming  conflicts can't appear any more.

TL;DR
Since the last update of @angular/material it is now possible to choose between the md or the mat prefix for components, directive etc.
For CSS classes they completly changed it to mat.

Now my question is:
Do we want to keep it like it is with the md prefix for components etc. and take thr mat prefix only for CSS or do we want to change completly to the mat prefix to prevent confusions?

Let's discuss @jmesserli @needToRoll

**Comment by needToRoll** [comment]
I do not really care, but in my opinion it is more elegant to use the same prefix all over.

It is a fact that I do not have as much frontend experience as you do.
Therefor I think, as our frontend-chef, this is your call.

Would you care to state your opinion?

**Comment by jmesserli** [comment]
Imo the move they did was useless but since they completely moved to `mat` for CSS, I think we should use that too.

**Comment by M4R1KU** [comment]
No it wasn't useless tough atleast for some people it wasn't.

I support your opinions too to change it to mat everywhere.
